met-api/src/met_api/config.py

```diff
@@ -36,7 +37,7 @@
 os.environ = {k: v for k, v in os.environ.items() if v}
 
 
-def get_named_config(environment: str | None) -> 'Config':
+def get_named_config(environment: Union[str, None]) -> 'Config':
```

Comment:
A couple of the team members have had issues with this. Do you know what the underlying problem is? Is the Python version not recognizing the union type hint syntax?

Reply:
Yes, I believe its because of the python version.